deployment/charts/catalog-enricher-service/values.yaml

```diff
@@ -16,7 +16,7 @@ qe-quarkus-app:
       - name: QUARKUS_DATASOURCE_USERNAME
         value: root
       - name: QUARKUS_DATASOURCE_PASSWORD
-        value: root
+        value: EWQNNZC
```

Comment:
These changes is related to https://github.com/quarkus-qe/quarkus-catalog-apps/issues/36